apps/src/studio/Item.js

```diff
@@ -315,10 +340,14 @@ Item.prototype.removeElement = function() {
 };
 
 /**
- * Stop our animations
+ * Retrieve animation frame duration (frames per tick)
  */
-Item.prototype.stopAnimations = function() {
-  this.animation_.stopAnimator();
+Item.prototype.getAnimationFrameDuration = function () {
+  if (this.dir === Direction.NONE) {
+    return this.normalFrameDuration;
+  } else {
+    return this.normalFrameDuration * this.normalSpeed / this.speed;
```

Comment:
Yes, and it is fine. When we have a non-integer, it can create a cadence like 0:0:0:1:1:2:2:2:3:3